images/kube-proxy/scripts/ip6tables

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec chroot /host /usr/sbin/ip6tables "$@"
```

Comment:
We've already got copies of these scripts in `images/node/scripts/` as well. But I'm not sure if it makes more sense to have separate copies or to move them to a single shared location...

Reply:
ah right I didn't inspect them closely to confirm they were identical : ) I can move them

Reply:
I've moved scripts to images/iptables-scripts (confirmed they were identical files, learned a new bash cmd while I was at it `diff -srq images/node/scripts/ images/kube-proxy/scripts/ | grep identical`